privacy/whonix.md

```diff
@@ -24,30 +24,25 @@ while the workstation is used for making AppVMs.
 
 ## Getting Started with Whonix
 
-To install Whonix, you must have a working Qubes machine already.
-
-*  [Install Whonix in Qubes](/doc/whonix/install/)
+*  [Installing Whonix in Qubes](/doc/whonix/install/)
 *  [Updating Whonix in Qubes](/doc/whonix/update/)
 
 ## Customizing, Reinstalling, & Uninstalling Whonix
 
 *  [Customizing Whonix](/doc/whonix/customize/)
-*  [Uninstall Whonix from Qubes](/doc/whonix/uninstall/)
+*  [Uninstalling Whonix from Qubes](/doc/whonix/uninstall/)
 
-*The following links are on Whonix's website and may be technical.*
+*The following pages are from Whonix's website.*
```

Comment:
What I meant to do was iterate that the links weren't on the Whonix website (they're on the Qubes website) and that instead the pages were written by the Whonix developers and located on their website. It's nitpicky but it's more correct.